crates/gpui/src/platform/windows/platform.rs

```diff
@@ -396,14 +392,110 @@ impl Platform for WindowsPlatform {
         false
     }
 
-    // todo!("windows")
     fn write_to_clipboard(&self, item: ClipboardItem) {
-        unimplemented!()
+        let wide_text: Vec<u16> = OsStr::new(item.text()).encode_wide().chain(Some(0)).collect();
+
+        let result = unsafe { OpenClipboard(None) };
+        if result.is_err() {
+            log::error!("OpenClipboard failed: {:#?}", unsafe { GetLastError() });
+            return;
+        }
+
+        let _ = unsafe { EmptyClipboard() };
+
+        // allocate memory to hold clipboard contents
+        // size is wide_text.len() * 2 because each character is 2 bytes (u16)
+        let handle = unsafe { GlobalAlloc(GLOBAL_ALLOC_FLAGS(0), wide_text.len() * 2).unwrap() };
```

Comment:
don't we have to allocate it as movable?

reading https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setclipboarddata
> If the hMem parameter identifies a memory object, the object must have been allocated using the function with the GMEM_MOVEABLE flag.

Reply:
This should be `GMEM_MOVEABLE`